setup.cfg

```diff
@@ -32,6 +32,7 @@ install_requires =
     pandas>=1.2.0
     requests
     joblib
+    pyarrow
```

Comment:
We need to document in the user-facing function that needs pyarrow that pyarrow is indeed needed for that function.